cdap-app-fabric/src/main/java/io/cdap/cdap/config/PreferencesService.java

```diff
@@ -155,7 +168,8 @@ private void setConfig(ProfileStore profileStore, PreferencesTable preferencesTa
     }
   }
 
-  private void deleteConfig(EntityId entityId) {
+  private void deleteConfig(EntityId entityId) throws Exception {
+    authorizationEnforcer.enforce(entityId, authenticationContext.getPrincipal(), Action.EXECUTE);
```

Comment:
Same here, how does InstanceId work?

Reply:
As discussed, auth extension can choose to ignore or enforce if it is instance id